helper/memory_calculator.go

```diff
@@ -94,17 +94,34 @@ func (m MemoryCalculator) Execute() (map[string]string, error) {
 			}
 		}
 
-		a, err := count.Classes(s)
+		staticAdjustment := 0
+		adjustmentFactor := uint64(100)
+		if adj, ok := os.LookupEnv("BPL_JVM_CLASS_ADJUSTMENT"); ok {
+			if strings.HasSuffix(adj, "%") {
+				if adjustmentFactor, err = strconv.ParseUint(strings.TrimSuffix(adj, "%"), 10, 32); err != nil {
+					return nil, fmt.Errorf("could not parse $BPL_JVM_CLASS_ADJUSTMENT %s as a percentage: %w", adj, err)
+				}
+			} else {
+				if staticAdjustment, err = strconv.Atoi(adj); err != nil {
+					return nil, fmt.Errorf("could not parse $BPL_JVM_CLASS_ADJUSTMENT %s as an integer: %w", adj, err)
+				}
+			}
+		}
+
+		a, err := count.Classes(appPath)
+
+		totalClasses := float64(j+a+staticAdjustment) * float64(adjustmentFactor) / 100.0
```

Comment:
Would you mind refactoring the single named env variables so this is more readable? I know that's an inherited problem, but it'll make this calculator more understandable when reading the code.